File: proyectos_c#/juegos/JuegoSnake/JuegoSnake/Pantalla.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JuegoSnake
{
    public class Pantalla
    {
        private int x, y, height, width, tamano;
        public Pantalla()
        {
            ////Size(369, 262)
            this.x = 0;
            this.y = 0;
            this.height = 262;
            this.width = 369;
            this.tamano = 10;
        }
        public void mantenerx(Cola snake)
            {
                if (snake.verX() < 0)
                {
                    snake.x = this.width - tamano;
                }
                else if (snake.verX() > this.width - tamano)
                {
                    snake.x = 0;
                }
            }

        public void mantenery(Cola snake)
            {
                if (snake.verY() < 0)
                {
                    snake.y = this.height - tamano;
                }
                else if (snake.verY() > this.height - tamano)
                {
                    snake.y = 0;
                }
            }
    }
}

File: proyectos_basic/juegos/SnakeBasic/SnakeBasic/Pantalla.vb
Public Class Pantalla
    Dim x, y, height, width, tamano As Integer
    Public Sub New()
        'Size(567, 285)
        Me.x = 0
        Me.y = 0
        Me.height = 285
        Me.width = 567
        Me.tamano = 10
    End Sub
    Public Sub mantenerx(snake As Cola)
        If (snake.verX() < 0) Then
            snake.x = Me.width - tamano
        ElseIf (snake.verX() > Me.width - tamano) Then
            snake.x = 0
        End If
    End Sub

    Public Sub mantenery(snake As Cola)

        If (snake.verY() < 0) Then
            snake.y = Me.height - tamano
        ElseIf (snake.verY() > Me.height - tamano) Then
            snake.y = 0
        End If
    End Sub
End Class
